### DISCUSSION TITLE
How can I create a FTP container?

### DISCUSSION THREAD

**Discussion by Martin-Andersen** [testcontainers/testcontainers-dotnet#1283] [Q&A]
I need a FTP test server. I found one at [testenvironment-docker](https://github.com/Deffiss/testenvironment-docker/tree/master/src/TestEnvironment.Docker.Containers.Ftp)

Can I use that packages? and how? 
If not then how can I build my own ftp test server?

**Comment by HofmeisterAn** [comment]
No, this is a different project. However, you can certainly use Testcontainers to create an FTP server instance and utilize it in your tests using the generic container builder API ([`ContainerBuilder`](https://dotnet.testcontainers.org/)). You can use any container image that provides an FTP server.
